surveys/management/commands/import_data.py

```diff
@@ -5,53 +5,54 @@
 from django.core.management.base import BaseCommand
 from django.conf import settings
 
+from surveys import models
+
 DB_CONN = 'postgresql://{USER}:{PASSWORD}@{HOST}:{PORT}/{NAME}'
 DB_CONN_STR = DB_CONN.format(**settings.DATABASES['default'])
 
+CENSUS_AREAS = {
+    'USA': ['1'],
+    'Pennsylvania': ['42'],
+    'Philadelphia': ['42101']
```

Comment:
The FIPS codes are stored in lists so that when we create new areas we can have them be groups of Census tracts (e.g. University City will be a list of tracts). I started simple here.

Reply:
Nice!